scout/server/blueprints/variant/utils.py

```diff
@@ -175,15 +176,16 @@ def update_variant_case_panels(case_obj: dict, variant_obj: dict):
     The case_obj should be up-to-date first. Call update_case_panels() as needed in context:
     to save some resources we do not call it here for each variant.
     """
-
+    variant_obj["case_panels"] = []
     variant_panel_names = variant_obj.get("panels") or []
-    case_panel_objs = [
-        panel
-        for panel in (case_obj.get("panels") or [])
-        if panel["panel_name"] in variant_panel_names
-    ]
-
-    variant_obj["case_panels"] = case_panel_objs
+    for panel in case_obj.get("panels") or []:
+        if panel["panel_name"] not in variant_panel_names:
+            continue
+        latest_panel_hgnc_genes: List[int] = store.panel_to_genes(
+            panel_name=panel["panel_name"], gene_format="hgnc_id"
+        )
+        if set(latest_panel_hgnc_genes).intersection(variant_obj["hgnc_ids"]):
```

Comment:
Nice! Is this faster than asking `not isdisjoint`?

Reply:
I don't know but I can check. I'm a bit afraid that collecting the latest panel genes for each variant in the variantS list might slow down too much, so I'd like to do some more testing of this PR

Reply:
Right, sounds wise. I think it has its given place on the variant page though.